rust/lance-encoding/src/encodings/logical/struct.rs

```diff
@@ -607,7 +607,20 @@ impl StructuralStructDecoder {
         should_validate: bool,
     ) -> Box<dyn StructuralFieldDecoder> {
         match field.data_type() {
-            DataType::Struct(fields) => Box::new(Self::new(fields.clone(), should_validate, false)),
+            DataType::Struct(fields) => {
+                let field_metadata = field.metadata();
+                if field_metadata
+                    .get("packed")
+                    .map(|v| v == "true")
+                    .unwrap_or(false)
```

Comment:
Also case insensitive (or replace with call to helper method)

Reply:
fixed.